CHANGELOG.md

```diff
@@ -4,6 +4,16 @@ The IronWeb SDK NPM releases follow standard [Semantic Versioning](https://semve
 
 **Note:** The patch versions of the IronWeb SDK will not be sequential and might jump by multiple numbers between sequential releases.
 
+## v3.4.1
+
+Encrypted search index feature. Adds methods to create a blind encrypted search index that allows for searching over encrypted data.
+
++ New namespace, `IronWeb.search`, that adds methods to create and initialize a blind search index and then methods for tokenizing data and search queries.
+  + A new blind search index can be created via the `IronWeb.search.createBlindSearchIndex(groupId: string): BlindIndex` to create a new search index that is accessible by everyone in the provided group.
+  + Initialize an existing search index via `IronWeb.search.initializeBlindSearchIndex(index: BlindIndex): InitializedSearchIndex` which takes the result of the `createBlindSearchIndex` call and returns an `InitializedSearchIndex` which contains methods for tokenizing data
+  + The `tokenizeData` method on `InitializedSearchIndex` creates an index for the provided plaintext data. The resulting `Uint32Array` is the index of all the trigrams in the provided plaintext.
+  + The `tokenizeQuery` method on `InitializedSearchIndex` tokenizes the provided search query into trigrams and returns a `Uint32Array` which can be compared against the result of `tokenizeData` to find matches.
+
```

Comment:
@BobWall23 @coltfred I could use some review on this changelog as I feel like I'm sort of winging it on my descriptions. I don't want to add too much content here and we should add the "Here's more info" and link to the content on the website that we'll eventually have.